.github/workflows/push.yml

```diff
@@ -85,14 +85,15 @@ jobs:
             ~/.cargo/registry
             ~/.cargo/bin
             ~/.cargo/git
-            target
+            ~/.cargo/.crates.toml
+            ~/.cargo/.crates2.json
           key: clippy-${{ runner.os }}-${{ hashFiles('rust-toolchain.toml') }}-${{ hashFiles('**/Cargo.lock') }}
           restore-keys: clippy-${{ runner.os }}-${{ hashFiles('rust-toolchain.toml') }}
 
-      - name: Install utils for caching
+      - name: Install cargo utils
         run: |
-          test -x ~/.cargo/bin/cargo-sweep || cargo install cargo-sweep
-          test -x ~/.cargo/bin/cargo-cache || cargo install cargo-cache --no-default-features --features ci-autoclean cargo-cache
+          cargo install cargo-sweep
+          cargo install cargo-cache --no-default-features --features ci-autoclean cargo-cache
```

Comment:
Given that we don't cache `target`, both of these are useless. Please just look at caching in monorepo

Reply:
Okay. I fixed it and simplified the cache key (similar to the monorepo.)